torchao/prototype/calibration_flow/static_quant.py

```diff
@@ -0,0 +1,107 @@
+"""
+Demo for static quantization flow
+"""
+import torch
+import copy
+
+from torch.ao.quantization.observer import MinMaxObserver, PerChannelMinMaxObserver
```

Comment:
it's kind of weird to import the old quantization flows from the new ones, maybe add a TODO to fix this in the future

Reply:
yeah sounds good